### DISCUSSION TITLE
background parameter in the peakAnnoEnrichment() function

### DISCUSSION THREAD

**Discussion by EraslanBas** [GreenleafLab/ArchR#1890] [Questions / Documentation]
Hi,

In the  peakAnnoEnrichment() function _background_  parameter is defined as "A string that indicates whether to use a background set of matched peaks to compare against ("bgdPeaks") or all peaks ("all")." May I ask how these background set of matched peaks are computed? I get very different enrichment results when I change this parameter between "bgdPeaks" and "all".

Thanks in advance.

**Comment by rcorces** [comment]
background peaks are regions matched by GC content and accessibility

https://github.com/GreenleafLab/ArchR/blob/f6c0388bd37023400794c9ae8562ad69e3ba9fd7/R/MatrixDeviations.R#L610-L699
